.github/workflows/tag-git.yml

```diff
@@ -99,6 +99,8 @@ jobs:
         with:
           app-id: ${{ secrets.GH_APP_ID }}
           private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
+          title: "Tag Git ${{ steps.generate.outputs.tag-name }} @${{ env.REV }}"
+          summary: "Tag Git ${{ steps.generate.outputs.tag-name }} @${{ env.REV }}"
           append-text: 'Tagged Git ${{ steps.generate.outputs.tag-name }}'
```

Comment:
Wasn't this already mentioning the tag name in the mirrored check run? I guess putting it in the name makes it more obvious as that will show up directly on the PR page and not just on the check run page, so the change is an improvement.